bal/variant/base_blockchain.py

```diff
@@ -85,9 +91,11 @@ def get_difficulty(self, a_block_chain):
             return latest_block['difficulty']
 
     def get_adjusted_difficulty(self, latest_block, a_block_chain):
-        prev_adjustment_block = a_block_chain[-1 * DIFFICULTY_ADJUSTMENT_INTERVAL]
+        prev_adjustment_block = a_block_chain[-1 *
+                                              DIFFICULTY_ADJUSTMENT_INTERVAL]
         time_expected = BLOCK_GENERATION_INTERVAL * DIFFICULTY_ADJUSTMENT_INTERVAL
-        time_taken = latest_block['timestamp'] - prev_adjustment_block['timestamp']
+        time_taken = latest_block['timestamp'] - \
```

Comment:
again it's very unreadable and untraceable what we do here with 2 different lines.